### PR TITLE
Home page design feedbacks

### PR DESCRIPTION
**Pull request by hoangviet1993**
- [x] Weird tint under nav-bar.
(As it turns out, the photo that I was using was a cropped screenshot and has a small white border at the top of the image. The issue disappears the moment I used the proper image that Jordan emailed me.)
- [x] Low-res "Get Involved" image (set a fixed width to always display the image at optimal resolution)
- [x] Low-res Oppia Foundation logo(set a fixed width to always display the image at optimal resolution)
- [x] Insert video into laptop screen
- [x] Inconsistent left and right page margins. Some images go all the way to the edge.
( I paid a lot more attention to ensure every single md-card has the intended paddings at all view-widths)
- [x] Fill buttons with corresponding links.
- [x] Click areas for buttons in navbar should not overlap with each other.
- [x] Ensure single breakpoint at narrowed width that changes 1 to 2 columns for everything.
(all md-cards now share the same breakpoint of switching between horizontal-->vertical alignment)

### CODE REVIEW

**Comment by hoangviet1993** [comment]
Desktop view-width (>1280px):
screencapture-localhost-8080-2018-09-06-18_55_08
Tablet view-width (600-1280px) (iPad):
screencapture-localhost-8080-2018-09-06-18_54_28
Mobile view-width(<600px) (Pixel2):
screencapture-localhost-8080-2018-09-06-18_53_02

**Comment by hoangviet1993** [comment]
With the added video, the page now weighs around 1.2MB. We can offload the added 328KB (the MP4 video) by hosting elsewhere like youtube or vimeo.

File: static/assets/css/stylesheet.css [hunk 1]
```diff
@@ -89,6 +89,16 @@ video {
   font: inherit;
   vertical-align: baseline;
 }
+video {
```
Comment (seanlip) [inline static/assets/css/stylesheet.css]: This seems a bit general. Maybe make it more specific -- give it a css class or something?

Comment (hoangviet1993) [reply static/assets/css/stylesheet.css]: Done!

**Comment by seanlip** [comment]
Yep all good. Thanks!

**Pull request merged**